app/assets/stylesheets/providers/search_results.scss

```diff
@@ -1,10 +1,11 @@
 // stylelint-disable selector-max-id
+@use 'sass:color';
```

Comment:
Should this be double quotes?
If yes, should our linter have picked this up?

Reply:
I've changed it to double quotes, I'm not sure about the linters TBH!